SU-HKKU/cardinal_service_profile
# Spotlight node and view improvements

# READY FOR REVIEW

# Summary
- Added a couple fields to spotlights, migrated field data and used those field in the views and displays.

# Need Review By (Date)
- asap

# Urgency
- medium-high

# Steps to Test
1. checkout this branch
1. `blt drupal:sync --site=cardinalservice`
1. create/view some spotlights
1. verify data still is there for existing spotlights.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
